t/lib/WTSI/NPG/HTS/Illumina/LogPublisherTest.pm

```diff
@@ -71,6 +74,43 @@ sub publish_logs : Test(4) {
   is_deeply($obj2->get_avu($ID_RUN),
             {attribute => $ID_RUN,
              value     => 17550}, "Inferred $ID_RUN metadata is present");
+
+  # Test that the pipeline central, postqc and product release logs are archived together with the others
+  Readonly::Scalar my $LOGS_COUNT => 43;
+  my @log_files =
+    ('_software_npg_20241107_bin_npg_pipeline_central_17550_20241101-132705-1566962201.definitions.json',
+      '_software_npg_20241107_bin_npg_pipeline_post_qc_review_17550_20241104-124525-1544617117.definitions.json',
+      '_software_npg_20241107_bin_npg_pipeline_central_17550_20241101-132705-1566962201.log',
+      '_software_npg_20241107_bin_npg_pipeline_post_qc_review_17550_20241104-124525-1544617117.log',
+      'product_release_20241101-132705-1247165951.yml',
+      'product_release_20241104-124525-1500421497.yml');
+  my $temp_dir = tempdir(CLEANUP => 1);
+  my $ref_name = '151211_HX3_18448_B_HHH55CCXX';
```

Comment:
$ref_name looks suspiciously similar to a short name for reference_name. Probably change to $rf_name or similar?